cumulus/client/relay-chain-rpc-interface/src/rpc_client.rs

```diff
@@ -588,6 +588,16 @@ impl RelayChainRpcClient {
 			.await
 	}
 
+	/// Get the minimum number of backing votes for a candidate.
+	pub async fn parachain_host_minimum_backing_votes(
```

Comment:
Is this even needed in cumulus? It should not.

Reply:
It doesn't compile without it, I see all runtime APIs have counterparts here.
I'm not sure why they're needed though

Reply:
I had similar question some time ago, and got this suggestion from @skunert: https://github.com/paritytech/cumulus/pull/2160#issuecomment-1562490741

Reply:
I think we can open a separate issue to discuss this further

Reply:
see: https://github.com/paritytech/polkadot-sdk/issues/1332